src/main/java/org/broadinstitute/hellbender/utils/bwa/BwaMemIndex.java

```diff
@@ -185,6 +230,7 @@ private static void loadNativeLibrary() {
         }
     }
 
+    private static native boolean createReferenceIndex(String referenceName, String indexPrefix, String algorithmName);
```

Comment:
In any case we could keep to separate private natives for both operations and provide only ```public``` method for those operations that we want the user to be able to perform.